Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs

```diff
@@ -105,26 +105,31 @@ public static IHtmlString ShowImages(this IPublishedContent item, string propert
         }
 
         /// <summary>
-        /// Works for MultipleMediaPicker where the image media is of type Image Cropper(as opposed to upload).
+        /// Works for MultipleMediaPicker where the image media is of type Image Cropper(as opposed to upload). Shows the image's urls in their cropped version.
         /// </summary>
         /// <param name="item"></param>
         /// <param name="cropAlias"></param>
         /// <param name="property"></param>
         /// <param name="recurse"></param>
+        /// <param name="prepend"></param>
         /// <param name="append"></param>
         /// <returns></returns>
-        public static IHtmlString ShowImageUrlCropped(this IPublishedContent item, string cropAlias, string property = "image", bool recurse = false, string append = "")
```

Comment:
That's true if we are upgrading from the old SuperHelper, and I guess we haven't used this new package on enough sites to be worried about breaking things when we upgrade nuget.

However, in future I think we need to be much more careful about introducing breaking changes to this package.